onepassword/cli.py

```diff
@@ -1,6 +1,7 @@
 import argparse
 import getpass
 import os
+import pyperclip
```

Comment:
This needs to be added to the install_requires in setup.py

Reply:
Right! Sorry :)